docs/source/pages/lightning.rst

```diff
@@ -118,85 +96,125 @@ If ``on_epoch`` is True, the logger automatically logs the end of epoch metric v
             self.valid_acc(logits, y)
             self.log('valid_acc', self.valid_acc, on_step=True, on_epoch=True)
 
+As an alternative to logging the metric object and letting Lightning take care of when to reset the metric etc. you can also manually log the output
+of the metrics.
+
+.. testcode:: python
+
+    class MyModule(LightningModule):
+
+        def __init__(self):
+            ...
+            self.train_acc = torchmetrics.Accuracy()
+            self.valid_acc = torchmetrics.Accuracy()
+
+        def training_step(self, batch, batch_idx):
+            x, y = batch
+            preds = self(x)
+            ...
+            batch_value = self.train_acc(preds, y)
+            self.log('train_acc_step', batch_value)
+
+        def training_epoch_end(self, outputs):
+            self.train_acc.reset()
+
+        def validation_step(self, batch, batch_idx):
+            logits = self(x)
+            ...
+            self.valid_acc.update(logits, y)
+
+        def validation_epoch_end(self, outputs):
+            self.log('valid_acc_epoch', self.valid_acc.compute())
+            self.valid_acc.reset()
+
+Note that logging metrics this ways will require you to manually reset the metrics at the end of the epoch yourself. In general we recommend logging
+the metric object to make sure that metrics are correctly computed and reset. Additionally, we highly recommend that the two ways of logging is not
+mixed as it can lead to wrong results.
```

Comment:
```suggestion
Note that logging metrics this way will require you to manually reset the metrics at the end of the epoch yourself. In general, we recommend logging
the metric object to make sure that metrics are correctly computed and reset. Additionally, we highly recommend that the two ways of logging are not
mixed as it can lead to wrong results.
```